### ISSUE TITLE
Comments left via the plugin hide line contents in AzDO preview

### ISSUE THREAD

**Issue by ottowayi** [ankitbko/vscode-pull-request-azdo#43]
- Extension version: 0.0.21
- VSCode Version: 1.55.2
- OS: Windows 10

Steps to Reproduce:

1. Checkout a PR in this plugin
2. Leave a comment on a line

Leaving a comment via this plugin in vscode causes the preview in AzDO to be missing all but the last character of the line the comment is set on

![image](https://user-images.githubusercontent.com/5274950/115774476-3b85eb00-a377-11eb-9ddb-1b98abfeba73.png)
![image](https://user-images.githubusercontent.com/5274950/115774342-12655a80-a377-11eb-9621-dc4c0d1c002c.png)

**Comment by ankitbko** [comment]
I did not understand it. Is the actual line `max_line_length=100` not present in AzDO UI? Have you commited and pushed your changes?

**Comment by ottowayi** [comment]
Yeah there are no changes locally.  This screenshot is from a PR that I checked out via the plugin and added the comment on line 36.  When I view the comment in AzDO you can see it only shows the last character (0) for the line I left the comment on.  It didn't modify the code, just whatever is used to show the comment location in the file.   It doesn't break anything, but makes it hard to review comments in AzDO because you lose the line that the comment was left on.

**Comment by ankitbko** [comment]
Adding a comment will not change the line itself. The AzDO UI will show what is the latest commit in the branch in remote. And VS Code will show whatever is the latest commit in local. Maybe you don't have the latest change? Could you do `git fetch` and then `git pull`.
